chihacknight/il-solar-map
# planned solar projects

* closes #42 
* partially implements #31 

Tasks to implement:

- [x] identify source files for planned projects from EIA, ABP and ILSFA
- [x] add source files to project and extract to csv files
- [x] combine planned projects into one file
- [x] update aggregates to include planned projects for `dg_small`, `dg_large`, `cs` and `utility`
- [x] add planned numbers to geojson and csv files
- [x] update map to toggle between energized and planned
- [x] update tables to show energized and planned
- [x] update about page text to include planned project info and counts

![Screenshot 2024-05-10 at 5 08 37 PM](https://github.com/chihacknight/il-solar-map/assets/919583/b0ce1d44-2fae-41a7-8b67-1faadaaadf36)

### Testing instructions
* browse the map and confirm you can toggle between energized and planned projects (planned should show up in purple). hovering over a geography should show a table with energized and planned projects
* confirm that the buttons selected are updated in the URL as you click them. copying a URL with options selected and pasting into a new tab/window should load the map with those options selected
* confirm on the about page that there is a new section on planned solar projects
* view the reference tables page and confirm they all include a set of columns for planned projects
